src/components/backdropscreensaver/plugin.js

```diff
@@ -25,12 +21,12 @@ define(['connectionManager'], function (connectionManager) {
                 Limit: 200
             };
 
-            var apiClient = connectionManager.currentApiClient();
+            const apiClient = connectionManager.currentApiClient();
             apiClient.getItems(apiClient.getCurrentUserId(), query).then(function (result) {
 
                 if (result.Items.length) {
 
-                    require(['slideshow'], function (slideshow) {
+                    import('slideshow').then(slideshow => {
```

Comment:
Can we avoid this import?

Reply:
Ah, we can leave these for later, it would make the diff even less understandable.